imap_processing/cdf/config/shared/hermes_default_global_cdf_attrs_schema.yaml

```diff
@@ -0,0 +1,278 @@
+CDF_Lib_version:
+  description: > 
+    Version of the CDF Binaries library used to generate the CDF File
+  default: null
+  derived: true
+  required: true # NOT Required in ISTP Guide (Derived)
+  validate: false
+  overwrite: false
+DOI:
+  description: > 
+    DOI is a persistent Unique Digital Identifier with the form
+    https://doi.org/<PREFIX>/<SUFFIX> with the <PREFIX> identifying the DOI
+    registration authority and the <SUFFIX> identifying the dataset. The DOI should point to
+    a landing page for additional information about the dataset. DOIs are typically created by
+    the SPASE naming authority or archive.
+  default: null
+  derived: false
+  required: true # NOT Required in ISTP Guide (Recommended)
```

Comment:
There are a lot of attributes here which are NOT required for ISTP, but are required for HERMES. I think I did a decent job adding comments on these cases. 

There are also som attributes that are marked required for HERMES, like Data_Level, which is not ISTP required, but it is used in our derivation functions for generating Logical_file_id.

Reply:
Ok, thank you! I think I'll leave the variables that are not required for ISTP in, since we probably will use some of them as well. But I will remove the obvious HERMES-specific ones and rename the file to be more generic.

Reply:
Is there a value in having multiple global_cdf_attrs schemas that can be mixed/matched as missions need?

For example, have an `ISTP_metadata_schema.yaml`, `SOLARNET_metadata_schema.yaml`, `IMAP_metadata_schema.yaml`, `SWxSOC_metadata_schema.yaml`, ... Where you would have ISTP / SOLARNET / external requirements able to be used by any mission and then each mission could pick and choose which individual metadata schemas they want to use and also add their own in their own repos so we don't blow up the main one with extra metadata for mission-specific things?

Reply:
Yes, this is something we've been working on is "layering" multiple YAML files together. https://swxsoc.readthedocs.io/en/latest/generated/api/swxsoc.util.schema.SWXSchema.html#swxsoc.util.schema.SWXSchema

This is how I envisioned it to be integrated, although I haven't had to time to fully do integration for HERMES yet. https://github.com/HERMES-SOC/hermes_core/blob/04013bd3748309f0cf88ff924f7ae12b30ee57b2/hermes_core/util/schema.py

Reply:
I'm also currently working on adding SOLARNET yaml files for FITS standards. Draft here: https://github.com/swxsoc/swxsoc/pull/5